### PR TITLE
release 10.7.0 beta2

### PR DESCRIPTION
**Pull request by jnweiger**
for https://github.com/owncloud/enterprise/issues/4449

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -2,27 +2,27 @@ def main(ctx):
   versions = [
 
     {
-      'value': '10.6.0',
-      'qa': 'https://download.owncloud.org/community/testing/owncloud-complete-20201216-qa.tar.bz2',
-      'tarball': 'https://download.owncloud.org/community/owncloud-complete-20201216.tar.bz2',
-      'tarball_sha': '3853bb528c1d4d274837faced9b3e27d343e9eadbb5cf692e5eeacb91083e529',
+      'value': '10.7.0beta2',
```
Comment (xoxys) [inline .drone.star:5]: Use `10.7.0-beta2`

**Review by xoxys — CHANGES_REQUESTED** [review]
Tarball does not exist

**Comment by xoxys** [comment]
Why is it called beta2? Was there a beta1?

**Comment by micbar** [comment]
> Why is it called beta2? Was there a beta1?

Yes, it was tagged but not built.

File: .drone.star (hunk 1 above)
Comment (jnweiger) [reply .drone.star]: thanks.
path fixed
hyphen added. (how do you remember, when a version number has to be without and when with a dash?)

Comment (micbar) [reply .drone.star]: > hyphen added. (how do you remember, when a version number has to be without and when with a dash?)
Dockerism: use a hyphen
oc10 -ism: don't use hyphens (historic reasons)

**Review by phil-davis — APPROVED** [review]
LGTM now

**Pull request merged**